onnxruntime/core/session/inference_session.cc

```diff
@@ -186,7 +186,10 @@ common::Status InferenceSession::RegisterCustomRegistry(std::shared_ptr<CustomRe
 
 common::Status InferenceSession::Load(std::function<common::Status(std::shared_ptr<Model>&)> loader, const std::string& event_name) {
   Status status = Status::OK();
-  auto tp = session_profiler_.StartTime();
+  TimePoint tp;
+  if (session_profiler_.FEnabled()) {
```

Comment:
Should this be IsEnabled() not FEnabled()?

Reply:
When I submitted the PR the method name was `FEnabled`. In the meantime it changed to `IsEnabled`. One more reason to do quick PR reviews :) I'll fix it and merge in the latest master.